charts/ccsm-helm/values.yaml

```diff
@@ -518,21 +518,25 @@ identity:
 
   # Keycloak configuration, for the keycloak dependency chart which is used by identity
   keycloak:
+    # Keycloak.service configuration, to configure the service which is deployed along with keycloak
     service:
+      # Keycloak.service.type can be set to change the service type.
+      # We use clusterIP for keycloak service, since per default LoadBalancer is used, which is not support on all cloud providers.
```

Comment:
```suggestion
      # We use clusterIP for keycloak service, since per default LoadBalancer is used, which is not supported on all cloud providers.
```